src/clue.js

```diff
@@ -1,14 +1,126 @@
 // ITERATION 1
 
+//randomize arr length to pick rnd
+// let testArr = [12,23,34]
+
+let selectRandom = (arr) => {
+
+    if (arr.length === 0) {
+        return undefined;
+    } else {
```

Comment:
Since you are returning within the first if statement, you don't have to make an else statement here